packages/docs/site/docs/main/how-to-create-a-theme-using-playground.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Build a WordPress Theme
+slug: /how-to-create-a-theme-using-playground
+---
+
+import ThisIsQueryApi from '@site/docs/\_fragments/\_this_is_query_api.md';
+
+# How to Build a WordPress Theme in WordPress Playground
+
+WordPress Playground is a powerful tool that lets you build, preview, and export WordPress themes—all in your browser or locally via `wp-now`.
+
+This guide walks you through the process step-by-step.
+
+---
+
+## 🚀 1. Creating a New Site
+
+### In the Browser
+
+1. Go to [https://playground.wordpress.net](https://playground.wordpress.net)
+2. Click **"Create New Site"**
+3. Choose WordPress and PHP versions if needed
+4. Click **"Launch"**
+
+### Locally with `wp-now`
+
+1. Install `wp-now`:
+
+    ```bash
+    npm create wp-now@latest
+    ```
+
+2. Navigate to your theme folder and run:
+
+    ```bash
+    wp-now
+    ```
+
+3. Access the local site at:
+
+    ```
+    http://localhost:8888
+    ```
+
+---
+
+## 🧰 2. Preinstall Theme-Building Tools
+
+Install the following plugins for theme development:
+
+-   [Create Block Theme](https://wordpress.org/plugins/create-block-theme/)
+-   [Gutenberg](https://wordpress.org/plugins/gutenberg/)
+-   [Theme Check](https://wordpress.org/plugins/theme-check/)
+
+Directly use playground links to load plugins: https://playground.wordpress.net/#%7B%22plugins%22:%5B%22hello-dolly%22,%22gutenberg%22%5D,%20%22login%22:%20true,%20%22landingPage%22:%20%22/wp-admin/plugins.php%22%20%7D
```

Comment:
This Playground link only installs the hello-dolly plugin again and the Gutenberg plugin but not the Create-Block-Theme plugin.